lib/stages/applyGains.cpp

```diff
@@ -224,96 +240,23 @@ void applyGains::apply_thread() {
 
         // Get the frequency index of this ID. The map will have been set in the first
         // validate_frame
```

Comment:
this comment is no longer accurate. 
```suggestion
        // Get the frequency index of this ID. The map will have been set by initialise
```